### DISCUSSION TITLE
Telescope's <C-x> mapping to create new notes

### DISCUSSION THREAD

**Discussion by davidsteinberger** [epwalsh/obsidian.nvim#315] [General]
For some reason the recently added `<C-x>` mapping does not work for me. 
I tried other mappings but none of them creates a new note. 
Is there some way to debug the telescope actions?

**Comment by epwalsh** [comment]
Hey @davidsteinberger, did you check `:messages` or `:Noice` for any errors? Was this from `:ObsidianQuickSwitch` or `:ObsidianSearch`? Is your telescope up-to-date?

**Reply by davidsteinberger** [reply]
Hm, I deactivated from my LazyVim setup _yanky_ (b/c it defines an action `<c-x>`) and my custom telescope config. I then invoke `:ObsidianSearch`, search for a non-existing text and upon an empty result invoke `<c-x>`. 
Noice shows a warning of `notify.warn telescope.nvim [telescope.actions.set.edit]: Nothing currently selected`.

**Reply by davidsteinberger** [reply]
Omg, now that I type this out I just realized my fatal mistake. The doc/readme is for _Head_, but I still run the latest released version. I just updated and everything works and makes sense now. Sorry!

**Reply by epwalsh** [reply]
Ahh yeup! I knew this would cause some confusion eventually 🤦‍♂️ I'll add a note in the README...

**Reply by epwalsh** [reply]
Added here: f9a3f03

Eventually I'd like to set up a separate `stable` branch which always points to the latest release. Then make that the default branch.
